### DISCUSSION TITLE
Welcome to Gestione-Allarme-Home-Assistant-Alarm Discussions!

### DISCUSSION THREAD

**Discussion by calas80** [Home-Assistant-Pro-Team/Gestione-Allarme-Home-Assistant-Alarm#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cangaroo82** [comment]
buongiorno 
esiste un video per l'installazione il progetto è bellissimo ma molto articolato nell'installazione

**Comment by docoscar** [comment]
Ciao,
progetto molto interessante e guida molto dettagliata. Sfortunatamente non ho chiaro come personalizzare i files all'inizio. Potresti spiegare in ordine cronologico i passaggi da fare per favore. Inoltre HA-Sip ha un problema simile: io sono dietro un fritzbox con un numero di telefono già attivo e configurato. Non riesco neanche a verificare se il funzionamento è corretto.

**Comment by Denfhor** [comment]
Ciao,
complimenti per il bellissimo progetto, ho seguito alla lettera i passaggi per l'istallazione su Home Assistant, ho creato una nuova plancia ma quando inserisco il codice sottostante mi da l'errore "No card type found", hai qualce suggerimento?,
grazie.
button_card_templates:
  no_background:
  styles:
  	  card:
	    - background-color: rgba(0,0,0,0.0)
	    - border-width: 0px
	  display:
	  tap_action:
		action: none
...........
title: xxxxx
views:
  - theme: Backend-selected
    title: test_alarm
    path: a
    badges: []
    cards:
...........
